product_replenishment_cost/models/product_product.py

```diff
@@ -28,14 +12,15 @@
 class ProductProduct(Model):
     _inherit = 'product.product'
 
-    @api.one
+    @api.multi
     @api.depends('product_tmpl_id.standard_price')
     def _get_replenishment_cost(self):
-        self.replenishment_cost = self.standard_price
+        for product in self:
+            product.replenishment_cost = product.standard_price
 
     replenishment_cost = fields.Float(
-        compute=_get_replenishment_cost, store=True,
```

Comment:
I had 2 reasons to remove store=True

1. The replenishment cost field defined in product_replenishment_cost module with store=True, creates the field in database with the value of product's standard_price and the value is not recomputed at the time of product_cost_incl_bom module installation.

2. Replenishment cost needs to be changed for each change in bom lines of product, when there is a change in their replenishment cost value and it did not work in that case as the method was not triggered for store=True. (when we change the BOM directly, it did not trigger the compute method and it does not allow multi-level BOM)